omx/gstomxvideodec.c

```diff
@@ -2639,10 +2639,8 @@ gst_omx_video_dec_handle_frame (GstVideoDecoder * decoder,
 static GstFlowReturn
 gst_omx_video_dec_finish (GstVideoDecoder * decoder)
 {
-  if (decoder->input_segment.rate > 0.0)
-    return gst_omx_video_dec_drain (decoder, TRUE);
-  else
-    return gst_omx_video_dec_drain (decoder, FALSE);
+  /* Drain without EOS flag to keep using the stream if needed */
+  return gst_omx_video_dec_drain (decoder, FALSE);
```

Comment:
With this change the is_eos argument of gst_omx_video_dec_drain function becomes unused, as well as all the logic related to the variable self->eos, we could remove all of that logic since now it's unused.

Also, did you tested the case when seeking to the end of video file in PLAYING and PAUSED state many times? With this change we would be draining the decoder component with an empty EOS buffer each time we reach the end of stream, so I would like to confirm that draining the component many times in that case won't break the decoder component.